runtime/near-vm-runner/src/tests/compile_errors.rs

```diff
@@ -285,10 +285,12 @@ pub fn test_stabilized_host_function() {
     (call $ripemd160 (i64.const 0) (i64.const 0) (i64.const 0)))
 )"#,
         )
+        // WasmtimeVM uses less gas than NearVM
+        .opaque_outcome()
```

Comment:
Most likely this difference is caused by a smaller module - if the module does not use memory, we do not "inject" it using Wasmtime